### PR TITLE
Feat : Add Property-Category Page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Screenshot (236)
Screenshot (237)
Screenshot (238)
Screenshot (239)

### CODE REVIEW

File: Product-Category/Family-category.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+    <!-- Navigation Bar For Tab and Mobile  -->
+
+    <div class="nav-bar nav-bar--hamburger">
+        <div class="nav-bar--logo">
+            <a href="/index.html"><img class="nav-bar__img--logo" src="/assets/image/HomePage-Images/Trendy-T Logo.png"
+                    alt="TRENDY-T"></a>
+        </div>
+        <div class="nav-bar--hamburger">
+            <img class="nav-bar__img--hamburger" src="/assets/image/HomePage-Images/Hamburger Menu.png" alt="Hamburger">
```
Comment (Kothalkarpriya) [inline Product-Category/Family-category.html:52]: Do use / before closing img tag

File: Product-Category/Family-category.html [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+        </div>
+    </div> -->
+
+    <!---Hamburger List -->
+    <div class="hamburger">
+        <div class="hamburger__header--wrapper">
+            <h1 class="hamburger__title">TRENDY-T</h1>
+            <img class="hamburger__close-btn" src="/assets/image/HomePage-Images/Hamburger Close.png" alt="Hamburger">
```
Comment (Kothalkarpriya) [inline Product-Category/Family-category.html:60]: every time where you used img tag

File: Product-Category/Kids-category.html [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+            <a href="/index.html">
+                <li class="hamburger__item">Home</li>
+            </a>
+            <a href="/Product-Page/product.html">
+                <li class="hamburger__item">Product</li>
+            </a>
+            <a href="/wishlist-page/wishlist.html">
+                <li class="hamburger__item">Wishlist</li>
+            </a>
+            <a href="">
```
Comment (Kothalkarpriya) [inline Product-Category/Kids-category.html]: do use # to make a dead link everytime

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]
Look into my comments and do make changes, then you can merge

File: Product-Category/Family-category.html (hunk 1 above)
Comment (SaurabhChawda) [reply Product-Category/Family-category.html:52]: Do I need to use / before closing img tag in javaScript?

File: Product-Category/product-category.css [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+  border: 2px solid var(--primary-color);
+}
+
+/***** sidebar *****/
+.sidebar {
+  visibility: visible;
+  position: fixed;
+  grid-area: sidebar;
+  overflow-y: scroll;
+  width:250px;
```
Comment (Namrata-J) [inline Product-Category/product-category.css:190]: try to use rems

File: Product-Category/product-category.css [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+main {
+  grid-area: main;
+  background-color: var(--secondary-bg-color);
+}
+
+/*card*/
+.card-product {
+  margin: 5rem 5rem;
+  display: grid;
+  gap: 20px;
```
Comment (Namrata-J) [inline Product-Category/product-category.css:253]: you could have used .1 or 1.2rem instead

**Review by Namrata-J — APPROVED** [review]

**Review by ShreyashMahajan — APPROVED** [review]
good to merge

**Pull request merged**